src/Service/Controllers/RestController.cs

```diff
@@ -6,12 +6,12 @@
 using System.Net.Mime;
 using System.Threading.Tasks;
 using Azure.DataApiBuilder.Service.Exceptions;
-using Azure.DataApiBuilder.Service.Models;
-using Azure.DataApiBuilder.Service.Services;
-using Azure.DataApiBuilder.Service.Services.OpenAPI;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Extensions;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.DataApiBuilder.Service.Models;
+using Microsoft.DataApiBuilder.Service.Services;
```

Comment:
How about naming it to DataApiBuilder.Core instead of DataApiBuilder.Service?

Reply:
Will make this change.